### ISSUE TITLE
MySims

### ISSUE THREAD

**Issue by Blayer98** [ValveSoftware/Proton#7451]
There seems to be a bug in the game (reported since 2014 via winehq but was never fixed) that causes the cursor to not pick up objects while building homes or while attempting to change your Sim's appearance by pressing either the left or right mouse button. I'm unable to make a post about it on the winehq page as Bugzilla does not want posts from people using third party wrappers (I can only presume Proton is one since it includes wine)

The log file that PROTON_LOG=1 generated tells me nothing in terms of the cursor issues I'm having so I'm completely stumped... I don't know if it's wine causing the issues or DXVK...

Proton Log (The log is quite big, I do apologize in advance!):
[steam-15313046569196978176.log](https://github.com/ValveSoftware/Proton/files/14074194/steam-15313046569196978176.log)

WineHQ Bug Report:
https://bugs.winehq.org/show_bug.cgi?id=37620

EDIT: Thanks to kisak if this was posted in the wrong GitHub issues page! Unfortunately MySims never got released on Steam, so I don't know if this is the best place...

**Comment by Blayer98** [comment]
Something I just noticed after I copied my Save data to my Steam Deck, the area to select an object or paint a wall are at the top of the screen, right in the middle (I’ll call this the select bug for easy reading), here's an example of this from a screenshot I took a few minutes ago:
![20240130164526_1](https://github.com/ValveSoftware/Proton/assets/6663677/3cce3764-055e-433e-9f8d-7473db11d34d)
As you can see, I can pick up the decoration because I was able to zoom the camera in close enough to grab it. Because the “hitboxes” seem to be generated at that exact spot, it’s the only way I can grab objects in Build Mode or paint objects or walls in Paint Mode.

The hitbox in question is located at the top middle of the screen. I cannot grab any other object unless said object is at the top middle of the screen like the roof decoration by zooming in and moving the camera backwards.

This most likely applies to CAS but I can’t test this as the camera never zooms in close enough for the Sim to reach the top of the screen.

I can confirm as well that this bug does not occur when making objects in the Workshop as you’re able to pick up all of the blocks, and you are able to paint said blocks you place down in the Workshop as well.

TL:DR: The select bug happens in the following areas:
Create A Sim (You cannot change the eyes, hairstyle, and clothing that your Sim has/wears)

When essence painting walls or floors in the interiors of any building.

When attempting to place building pieces down or change the look of them while building/re-modelling a building.

Where this bug does not occur:
When creating new objects/completing tasks in the Workshop (Both essence painting and placing blocks down)

When placing objects or moving objects around while decorating the interior of a building.

**Comment by Blayer98** [comment]
This issue has now been fixed in DXVK, just have to wait until the changes get added to Proton Experimental. :)

https://github.com/doitsujin/dxvk/commit/ab3593185f45971977b956323c0a60f66f9f35e7

**Comment by Blayer98** [comment]
Sadly, the latest Proton Experimental update that released today doesn’t yet include the fix for this game, will have to wait for the next update :)

**Comment by Blayer98** [comment]
As of the 26th February, this issue has now been added into Proton Experimental and this issue is fixed.

**Comment by powerpuffboysz** [comment]
i think we should reopen the issue for mysims proton compatability. it alongside mysims kingdom were rereleased on steam and have major controller detection issues
